### PR TITLE
Fix client trimHeader crash for header only data

### PR DESCRIPTION
**Pull request by foolenough**
If data is empty after remove header, return.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -333,6 +333,9 @@ func (c *Client) cleanup(ch chan *Event) {
 
 func trimHeader(size int, data []byte) []byte {
 	data = data[size:]
```
Comment (FatalReadError) [inline client.go:335]: good point on this. what if data is nil or len(data) is < size?

Comment (purehyperbole) [reply client.go:335]: Good question. I think that usecase is covered by https://github.com/r3labs/sse/blob/master/client.go#L283 and the `bytes.HasPrefix` checks before its invoked.

**Comment by purehyperbole** [comment]
Hey @foolenough thanks for the contribution. Is there any chance you could exclude the changes to the `README.md` and `go.mod` please? Happy to merge once thats in place. You may find it easier to create a branch on your repo and open a new PR?

**Comment by frederikhors** [comment]
@foolenough are you still there?

**Comment by codewinch** [comment]
Want me to re-submit?

**Pull request closed without merging**